Add postmortem of utility score bug (#1649)
helium/docs: devblog/2023-10-18-utility-score-bug-post-mortem.mdx

---
layout: post
title: MOBILE Utility Score Bug Post Mortem (Pre HIP-88)
date: 2021-10-18
hide_table_of_contents: true

> authors: [cokes518]

Reviewer: Let's add you to the authors table. I can push it to this branch.

Author: Oh oops, sorry had copied this from another one

---
layout: post
title: MOBILE Utility Score Bug Post Mortem (Pre HIP-88)
date: 2023-10-18
hide_table_of_contents: true
authors: [noah]
---

&nbsp;

# MOBILE Utility Score Bug Post Mortem (Pre HIP-88)

> This post-mortem describes the events leading to an incorrect A portion of the Utility Score for the
> MOBILE SubNetwork, what happened, and what the team is doing moving forward.

Reviewer 2: ```suggestion
This post-mortem describes the events leading to an incorrect `A` portion of the DAO Utility Score for the
MOBILE subnetwork, what happened, and what the team is doing moving forward.
```

<!--truncate-->

## Timeline

> On July 21st, 2023, MOBILE Subscriber Mapping went live. This introduced a new rewardable entity
> into the MOBILE ecosystem.

Reviewer 2: ```suggestion
On July 21st, 2023, MOBILE Subscriber Mapping launched. This introduced a new rewardable entity
into the MOBILE ecosystem.
```

> Helium Oracles are powered by a data pipeline that runs over Proof of Coverage and network usage
> data. After generating rewards shares, the lifetime rewards for each entity are calculated and
> inserted into a Postgres database. When a user initiates a claim, the Oracle service then reads the
> values from the database and signs the claim transaction to set the lifetime rewards for that entity
> on-chain.

Reviewer 2: ```suggestion
Helium Oracles are powered by a data pipeline that runs over Proof-of-Coverage and network usage
data. After generating rewards shares, the lifetime rewards for each entity are calculated and
inserted into a Postgres database. When a user initiates a claim, the Oracle service then reads the
values from the database and signs the claim transaction to set the lifetime rewards for that entity
on-chain.
```
